evap/student/templates/student_vote.html

```diff
@@ -51,7 +51,26 @@
             </small>
         </div>
     {% endif %}
-    <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
+    <h3 class="mb-3">
+        {{ evaluation.full_name }} ({{ evaluation.course.semester.name }})
+
+        <div class="btn-switch btn-switch-light float-end">
+            <div class="btn-switch-label my-auto">{% trans "Questionnaire language" %}</div>
+            <div class="btn-switch btn-group">
+                {% for language_code, language_name in languages %}
+                    {% csrf_token %}
+                    <a data-bs-toggle="tooltip"
+                       data-bs-placement="bottom"
+                       title="{{ language_name }}"
+                       role="button"
+                       href="{% url 'student:vote' evaluation.pk %}?language={{ language_code }}"
```

Comment:
This should probably be a POST-form, not a GET-anchor. (Currently, this wouldn't need the CSRF token)

Reply:
Ah, I assumed that this would save the language so that if a user came back later to the same evaluation, they would still go back to the language they last selected.

If we keep the language stateless, the anchor is fine here, but then we don't need the csrf_token above.

Reply:
we should save the language in localstorrage but not on the server

Reply:
csrf token is still there